### PR TITLE
Helm chart

### PR DESCRIPTION
**Pull request by rporres**
Deployed using manually built images from PR #133 (with below Caddyfile redirection added by hand). This PR must not be merged until that one is.

http://srcd.run
http://www.srcd.run

```
helm install -n landing . --set "\
image.landingSlackin.slackApiToken=TOKEN,\
ingress.hosts={srcd.run,www.srcd.run},\
ingress.globalStaticIpName=landing-srcd-run,\
image.registry=quay.io/rporres,\
image.tag=healthcheck-api
"

### CODE REVIEW

File: helm-charts/landing/values.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+replicaCount: 1
+
+# If provided, these will be used by the deployment
+# nodeSelector: XXXXXXX
+
+image:
+  # same tag for everyone as it is how landing repo Makefile works
+  tag: master
+  pullPolicy: IfNotPresent
+  repository: quay.io/srcd
```
Comment (smeruelo) [inline helm-charts/landing/values.yaml]: Nit-picking: Actually, the name of the repository is quay.io/srcd/landing, or quay.io/srcd/landing-api,.. I'd either use 3 different repository keys, or rename this to "registry".

Comment (smeruelo) [reply helm-charts/landing/values.yaml]: Also, do we really want to use 3 different repositories for these images? As an alternative, we could use "api" or "slacking" as part of the tag, together with the version, and this way we would have all 3 different images inside the same repository (this is the current approach, I think). It's not beautiful, but our license allows just 50 repository so using 3 for one single website seems like a waste.

Comment (rporres) [reply helm-charts/landing/values.yaml]: You're right. I will change the yaml key name.

As for the comment, I don't think we have a limit on the number of public repositories.

Comment (smeruelo) [reply helm-charts/landing/values.yaml]: You're right! Definitely better in separate repos then :)

File: helm-charts/landing/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          readinessProbe:
+            tcpSocket:
+              port: {{ .Values.service.landingSlackin.internalPort }}
+            initialDelaySeconds: 5
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
+        - name: {{ .Values.image.landingApi.name }}
+          image: "{{ .Values.image.registry }}/{{ .Values.image.landingApi.name }}:{{ .Values.image.tag }}"
```
Comment (smeruelo) [inline helm-charts/landing/templates/deployment.yaml]: I almost forgot! Please make `.Values.image.tag` mandatory.

Comment (rporres) [reply helm-charts/landing/templates/deployment.yaml]: I will create a new PR for this.

**Pull request merged**